container/ci/serverconfig.json

```diff
@@ -12,6 +12,9 @@
       }]
     }
   ],
+  "features": {
+    "regressionResultMsgs": true
```

Comment:
just note that this intends to be a temporary flag; once the feature is vetted, will remove flag from UI code and permanently activate feature (thus CI on localhost not using container will pass too)